prisma/migrations/20240201042705_init/migration.sql

```diff
@@ -0,0 +1,156 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Specialist` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `Therapy` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- CreateEnum
+CREATE TYPE "Gender" AS ENUM ('MALE', 'FEMALE');
+
+-- CreateEnum
+CREATE TYPE "FormatOfWork" AS ENUM ('OFFLINE', 'ONLINE', 'BOTH');
+
+-- DropTable
+DROP TABLE "Specialist";
+
+-- DropTable
+DROP TABLE "Therapy";
+
+-- CreateTable
+CREATE TABLE "specialists" (
+    "id" UUID NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "fullName" VARCHAR(128) NOT NULL,
+    "gender" "Gender" NOT NULL,
+    "yearsOfExperience" SMALLINT,
+    "formatOfWork" "FormatOfWork" NOT NULL,
+    "treatmentTypes" TEXT[] DEFAULT ARRAY[]::TEXT[],
+    "price" SMALLINT NOT NULL DEFAULT 0,
+    "isFreeReception" BOOLEAN NOT NULL,
+    "description" TEXT NOT NULL,
+    "phone" VARCHAR(15),
+    "email" VARCHAR(64),
```

Comment:
It is only part of email can have a length of 64 - https://emaillistvalidation.com/blog/demystifying-email-validation-understanding-the-maximum-length-of-email-addresses/